test/run-spec-wasm2c.py

```diff
@@ -173,6 +187,31 @@ def _CacheModulePrefixes(self):
                 self.module_prefix_map[name_idx] = name
                 self.unmangled_names[name_idx] = command['as']
 
+    def _WriteModuleInstanceImports(self, command, uninstantiable):
```

Comment:
The job of this function seems to be create the call to the init function?    Does it need to be renamed?  Maybe `_WriteModuleInitCall` or something like that?

Reply:
Done.